### PR TITLE
CMS API: Fix API Docs

### PR DESCRIPTION
**Pull request by jlledom**
**What this PR does / why we need it**:

The current API Docs for the CMS API have some mistakes. Basically unused parameters and some required params not marked as required.

**Which issue(s) this PR fixes** 

[THREESCALE-8991](https://issues.redhat.com/browse/THREESCALE-8991)

### CODE REVIEW

**Review by thalesmiguel — APPROVED** [review]

**Comment by jlledom** [comment]
I'm gonna merge this without a second review, it's simple and it only affects documentation. In fact, these are the OpenAPI v1 docs which we plan to remove soon.

**Pull request merged**
